File: model/src/main/java/com/sealinkin/cost/model/Cost_OtherSetModel.java
package com.sealinkin.cost.model;

import com.sealinkin.cost.po.Cost_OtherSet;

public class Cost_OtherSetModel extends Cost_OtherSet {
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	private String enterpriseNo;
	private String costNo;
	private String costNoText;
	
	public String getEnterpriseNo() {
		return enterpriseNo;
	}
	public void setEnterpriseNo(String enterpriseNo) {
		this.enterpriseNo = enterpriseNo;
	}
	public String getCostNo() {
		return costNo;
	}
	public void setCostNo(String costNo) {
		this.costNo = costNo;
	}
	public String getCostNoText() {
		return costNoText;
	}
	public void setCostNoText(String costNoText) {
		this.costNoText = costNoText;
	}
}

File: wms/src/main/webapp/system/app/model/cost/cost_OtherSetModel.js
/**
 * 模块名称：杂项费用维护
 * 模块编码：B203
 * 创建：hcx 
 */
Ext.define('cms.model.cost.cost_OtherSetModel', {
    extend: 'Ext.data.Model',
    idProperty:'enterpriseNo,cost_no',
    fields: [
        {name:'enterpriseNo'},
		{name:'costNo'},
		{name:'costName'},
		{name:'remark'},
		{name:'rgstName'},
		{name:'rgstDate'},
		{name:'updtName'},
		{name:'updtDate'},
		{name:'createFlag'},

		{name:'costNoText'}
    ]
});